docs/plugins/page-visibility-tracker.md

```diff
@@ -0,0 +1,190 @@
+  </tr>
+</table>
+
+**Note:** the reference to `options` refers to passed configuration [options](#options).
+
+### New pageview events
+
+If the page's visibility state changes from `hidden` to `visible` and the session has timed out. A pageview is sent instead of a change event.
+
+<table>
+  <tr valign="top">
+    <th align="left">Field</th>
+    <th align="left">Value</th>
+  </tr>
+  <tr valign="top">
+    <td><a href="https://developers.google.com/analytics/devguides/collection/analyticsjs/field-reference#hitType"><code>hitType</code></a></td>
+    <td><code>'pageview'</code></td>
+  </tr>
+</table>
+
+
+## Improving session duration calculations
+
+While not a feature enabled by default, the `pageVisibilityTracker` plugin is also capable of dramatically improving the accuracy of session duration calculations.
+
+Session duration in Google Analytics is defined as the amount of time between the first interaction hit and the last interaction hit within a single session. That means that if a session only has one interaction hit, the duration is zero, even if the user was on the page for hours!
+
+The `pageVisibilityTracker` plugin defaults to sending visibility state change events as [non-interaction](https://developers.google.com/analytics/devguides/collection/analyticsjs/events#non-interaction_events) hits (for the `hidden` event), but you can customize this to make all hits interactive and since most sessions will contain visibility state change events, your sessions durations will become much more accurate.
+
+The downside of this approach is it will alter your bounce rate. However, for many users, this trade-off is worth it since bounce rate can be calculated in other ways (e.g. with a custom segment).
+
+The examples section below includes a code sample showing [how to make all events interaction events](#ensuring-all-events-are-interaction-events).
+
+## Methods
+
+The following table lists all methods for the `pageVisibilityTracker` plugin:
+
+<table>
+  <tr valign="top">
+    <th align="left">Name</th>
+    <th align="left">Description</th>
+  </tr>
+  <tr valign="top">
+    <td><code>remove</code></a></td>
+    <td>Removes the <code>pageVisibilityTracker</code> plugin from the specified tracker, removes all event listeners from the DOM, and restores all modified tasks to their original state prior to the plugin being required.</td>
+  </tr>
+</table>
+
+For details on how `analytics.js` plugin methods work and how to invoke them, see [calling plugin methods](https://developers.google.com/analytics/devguides/collection/analyticsjs/using-plugins#calling_plugin_methods) in the `analytics.js` documentation.
+
+## Examples
+
+### Ensuring all events are interaction events
+
+This example uses the `fieldsObj` option to set the default `nonInteraction` value for all hits to `false`, which overrides the plugin's default `nonInteraction` value of `true` for hidden events.
```

Comment:
perhaps a quick comment about doing this to improve session duration calc and link to section above.